web/src/demo_organizations_ui.ts

```diff
@@ -108,19 +144,11 @@ export function do_convert_demo_organization(): void {
 }
 
 export function handle_demo_organization_conversion(): void {
-    $(".convert-demo-organization-button").on("click", () => {
+    $(".demo-organization-warning").on("click", ".convert-demo-organization", () => {
         do_convert_demo_organization();
     });
 
-    // Treat Enter with convert demo organization link as a click.
-    $(".demo-organization-warning").on(
-        "keyup",
-        ".convert-demo-organization-button[role=button]",
-        function (e) {
-            e.stopPropagation();
-            if (keydown_util.is_enter_event(e)) {
-                $(this).trigger("click");
-            }
-        },
-    );
```

Comment:
Is this enter-acts-like-click behavior present elsewhere? I don't think it's very important -- it's kinda hard to tab to the component -- so it's mostly a question of whether our banner component already does this.